### PR TITLE
[dnm actually, removing some changes for pr to main]Changes movespeed to pixel movement pixel lengths initial update

### PR DESCRIPTION
**Pull request by kevinz000**
DOES NOT SUPPORT lengths bigger than a mob's step size (coming in next PR), and DOES NOT animate the movement (hopefully coming in the next PR)!
DOES NOT change how movespeed is balanced in terms of net tiles moved per second. movement_delay() still exists and is being used.

### CODE REVIEW

**Comment by kevinz000** [comment]
will memo you when this is done, refactoring slowdown

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1075,7 +1075,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	if(H.movement_type & FLYING)
 		flight = 1
 
-	if(H.has_gravity())
+	if(H.has_gravity())kjlj
```
Comment (ninjanomnom) [inline code/modules/mob/living/carbon/human/species.dm]: gj

File: code/__DEFINES/configuration.dm [hunk 2]
```diff
@@ -1,5 +1,6 @@
 //config files
 #define CONFIG_GET(X) global.config.Get(/datum/config_entry/##X)
+#define CONFIG_GET_DATUM(X) global.config.ReturnEntry(/datum/config_entry/##X)
```
[the remark this answers is not in the document]
Comment (kevinz000) [reply code/__DEFINES/configuration.dm]: Returns the datum for a config entry, as the movespeed stuff caches that. Just caching the speed means that any changes to the entry won't actually work as the old cached speed would be used instead.

**Comment by ninjanomnom** [comment]
Bring this over to the new branch

**Pull request closed without merging**